cms/djangoapps/contentstore/views/course.py

```diff
@@ -815,6 +819,22 @@ def course_info_update_handler(request, course_key_string, provided_id=None):
     # can be either and sometimes django is rewriting one to the other:
     elif request.method in ('POST', 'PUT'):
         try:
+            if settings.FEATURES.get('NOTIFICATIONS_ENABLED', False) and request.method == 'POST':
+                # only send bulk notifications to users when there is
+                # new update/announcement in the course.
+
+                # get the notification type.
+                notification_type = get_notification_type(u'open-edx.studio.announcements.new_announcement')
+                course = modulestore().get_course(course_key, depth=0)
+                notification_msg = NotificationMessage(
+                    msg_type=notification_type,
+                    namespace=unicode(course_key),
+                    payload={'course_name': course.display_name}
```

Comment:
Can you also add a 'version' field here of '1' like you did in your other PR. I can't remember if it was called 'version' or '_version'. Thx.